api/src/test/java/uk/gov/dhsc/htbhf/hmrc/HMRCIntegrationTests.java

```diff
@@ -128,7 +134,21 @@ private void assertResponseCorrect(ResponseEntity<EligibilityResponse> response,
         assertThat(response.getStatusCode()).isEqualTo(OK);
         EligibilityResponse eligibilityResponse = response.getBody();
         assertThat(eligibilityResponse).isNotNull();
-        assertThat(eligibilityResponse).isEqualTo(expectedResponse);
+        assertEligibilityResponse(expectedResponse, eligibilityResponse);
+    }
+
+    // checks that two eligibility responses are equal whilst ignoring the order of the children
+    private void assertEligibilityResponse(EligibilityResponse expectedResponse, EligibilityResponse eligibilityResponse) {
+        assertThat(eligibilityResponse).isEqualToIgnoringGivenFields(expectedResponse, "children");
+        assertChildren(expectedResponse.getChildren(), eligibilityResponse.getChildren());
+    }
+
+    private void assertChildren(List<ChildDTO> expected, List<ChildDTO> actual) {
+        if (expected == null) {
```

Comment:
Do you need this null check when you're only using it in one place where it isn't null?

Reply:
The other option would be to have different assert methods, one with children and one without. It would require updating three other assert methods. I like the assertions methods being generic right now.